### ISSUE TITLE
Question about the training process

### ISSUE THREAD

**Issue by ithieund** [VinAIResearch/PhoBERT#34]
Hi @datquocnguyen 
Can you share some information about the training process?
For example:

- How many samples per batch?
- Which optimizer did you use and its params?
- Did you use learning rate schedule? its params?
- What is the result at the end? ex: training loss and accuracy

Thanks you so much.

**Comment by datquocnguyen** [comment]
Please see our paper: https://aclanthology.org/2020.findings-emnlp.92/

**Comment by ithieund** [comment]
Thanks but I can not find information about the final training loss.

**Comment by datquocnguyen** [comment]
I did not care about the training loss, thus I did not save that information.

**Comment by ithieund** [comment]
OK thank you @datquocnguyen
